### ISSUE TITLE
importing `siuba.data` fails

### ISSUE THREAD

**Issue by NightlyHerb** [machow/siuba#326]
Hello, on examples in [Quickstart](https://siuba.readthedocs.io/en/latest/intro.html) there seems to be an error when importing `siuba.data`:
(Edit: same error when `from siuba.data import mtcars`)
```py
>>> import siuba.data
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\su\Documents\Projects\misc\siuba-test\lib\site-packages\siuba\data\__init__.py", line 24, in <module>
    cars_sql = _LazyTbl(
  File "C:\Users\su\Documents\Projects\misc\siuba-test\lib\site-packages\siuba\sql\verbs.py", line 219, in __init__
    self.ops = [sql.Select([self.tbl])] if ops is None else ops
TypeError: __init__() takes 1 positional argument but 2 were given
>>>
```

Here are my laptop settings, I don't think it's useful but just in case:
OS: Windows 10
Python versions: Python 3.9.2 (latest conda) and Python 3.9.4 (installed with Scoop)

Thank you!

**Comment by machow** [comment]
Ah, thanks for raising--SqlAlchemy 1.14 released recently and had some backwards incomparable changes. Downgrading to 1.13.24 should fix it. Let me take a quick look at getting things compatible with the new version..

**Comment by machow** [comment]
Alright--I've got a version running with 1.4 passing tests locally! There's a bit left to do w/ making sure things still run on 1.3, and cleaning up some methods that 1.4 deprecated. Will take another pass over it / try to merge in the next week!

**Comment by machow** [comment]
Oof, okay--should be resolved in v0.0.25!
